src/pages/Transfer.vue

```diff
@@ -203,7 +203,7 @@ export default {
 
       const signedTx = await this.$store.dispatch('signTransaction', {
         transaction: {
-          fee: BigNumber(1).shiftedBy(-MAGNITUDE),
+          fee: BigNumber(15000 + (100 * 20)).shiftedBy(-MAGNITUDE),
```

Comment:
@edwarddikgale This variable is taken from here: https://github.com/aeternity/protocol/blob/master/consensus/consensus.md#gas. I would like to calculate min gas more carefully, but it is difficult with the current architecture. I can improve constants handling later with the support of different transaction types or even just to use code from SDK.

Reply:
Ok I got it and if you create a reminder for yourself in Pivotal to refactor this later we can approve it

Reply:
I have rebased this PR to solve conflicts and at the same time explained how min transaction fee is generated in 7cc1b4fe8ca23b3f310a67ca2cbf8e443c1f896a